src/hl7v2/schema.clj

```diff
@@ -49,6 +50,27 @@
          (drop (if (node-attrs node) 2 1))
          (into []))))
 
+(defn node-annotation [node]
+  (letfn [(rm-diacritics [s]
+            (when s
+              (-> s
```

Comment:
Esa flecha -> qué es??

Reply:
`(letfn` is the same thing as `(let` but only for functions, and the functions are local to the enclosing function, so they are temporary, as you say. This is a common practice in clojure used to reduce the footprint of the function. An alternative is to extract the function as a top-level form of the namespace, but this is discouraged when the function is a helper/utility not meant to be used outside the namespace, or not at the same level of abstraction of the namespace (SLA).

Reply:
As for the question about the `(->` thread left macro, it pipes the value through the functions, passing the result of each function as the first param to the next function. Try to understand this macro, because it is frequently used in Clojure.